### PR TITLE
Added Swift language support

### LINKED ISSUE #12: add grammar for swift
see [contributing instructions](https://github.com/asottile/babi-grammars#internalcontributing-instructions) if you'd like to work on this

### PR DESCRIPTION
**Pull request by ClasherKasten**
Closes #12

### CODE REVIEW

File: testdata/source_swift.swift [hunk 1]
```diff
@@ -0,0 +1,6 @@
+func say_hello(name: String) -> Void {
+    print("Hello \(name)")
+}
+
+say_hello(name: "asotille")
+
```
Comment (asottile) [inline testdata/source_swift.swift:6]: you can just copy paste my name :(

File: bin/download-syntax [hunk 2]
```diff
@@ -212,9 +212,12 @@ REPOS = (
     ),
     Repo(
         name='microsoft/vscode',
-        version='b37252c1823',
+        version='df15c8e832a',
         license_path='LICENSE.txt',
-        grammars=('extensions/java/syntaxes/java.tmLanguage.json',),
+        grammars=(
+            'extensions/java/syntaxes/java.tmLanguage.json',
+            'extensions/swift/syntaxes/swift.tmLanguage.json',
```
Comment (asottile) [inline bin/download-syntax:219]: generally this should come from the source repository -- even the `java` one probably shouldn't be coming from there

usually you can find the source repository in a "comment" at the top of the microsoft one

Comment (ClasherKasten) [reply bin/download-syntax:219]: Problem with that is that i already tried the original source (`textmate/swift.tmbundle`) earlier, but  when i updating the grammars it replaces `HEAD` to ``. But i can try it.

**Review by asottile — APPROVED** [review]

**Pull request merged**